test/engine/TextIndexScanForWordTest.cpp

```diff
@@ -87,6 +87,22 @@ std::string docsFileContent = createDocsFileLineAsString(4, firstDocText) +
 std::pair<std::string, std::string> contentsOfWordsFileAndDocsFile = {
     wordsFileContent, docsFileContent};
 
+// Return a `QueryExecutionContext` from the turtle `kg` (see above) that has a
+// text index that contains the literals from the `kg` as well as the
+// `contentsOfWordsFileAndDocsFile` (also above). The metrics used for the text
+// scores can be specified.
+auto getQecWithTextIndex(
```

Comment:
A personal preference of mine is to put an asterisk after the `auto*`.
This makes it clear that we get a raw pointer and not some sort of smart pointer or something. (Or just specify `QueryExecutionContext*` directly.)

Reply:
I deliberately leave this for now, as it is
"a raw pointer with static storage duration" which is really hacky, so I would leave the `auto` 
to not have to change this place, should this ever become i.e. something like `unique_ptr<QECThatOwnsItsIndex>`.